marklogic-data-hub-central/ui/src/pages/Overview.test.tsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import {Router} from "react-router";
 import {createMemoryHistory} from "history";
+import userEvent from "@testing-library/user-event";
 import {render, fireEvent} from "@testing-library/react";
```

Comment:
I'm also seeing this import for fireEvent repeated on line 10 and my testing is complaining, remove the one on line 10?

Reply:
I see the test file getting fireEvent from two different places, /react and /dom.

Reply:
fixed